### PR TITLE
Update documentation of opcodes

### PR DESCRIPTION
**Pull request by amityadav0**
- Update documentation of opcodes

### CODE REVIEW

File: packages/algorand-js/src/interpreter/opcode-list.ts [hunk 1]
```diff
@@ -153,10 +164,12 @@ export class Mul extends Op {
 
 // pushes argument[N] from argument array to stack
 export class Arg extends Op {
-  readonly _arg;
+  readonly _arg: Uint8Array;
 
   /**
-   * @param args words list extracted from line: {argument number}
+   * Description: Gets the argument value from interpreter.args array.
+   * store the vale in _arg variable
```
Comment (ratik21) [inline packages/algorand-js/src/interpreter/opcode-list.ts:171]:
```suggestion
   * store the value in _arg variable
```

File: packages/algorand-js/src/interpreter/opcode-list.ts [hunk 2]
```diff
@@ -183,7 +196,8 @@ export class Bytecblock extends Op {
   readonly interpreter: Interpreter;
 
   /**
-   * @param args words list extracted from line: {byte block}
+   * Description: Store blocks of bytes in bytecblock
+   * @param args Expected arguments: [byte block]
```
Comment (ratik21) [inline packages/algorand-js/src/interpreter/opcode-list.ts:200]: What does `[byte block]` mean? Expected arguments is an array of bytes. (eg. `bytecblock "a" "b" "c"`)

File: packages/algorand-js/src/interpreter/opcode-list.ts [hunk 3]
```diff
@@ -1144,7 +1205,8 @@ export class Branch extends Op {
   readonly interpreter: Interpreter;
 
   /**
-   * @param args words list extracted from line: {branch name}
+   * Description: Sets `label` according to arguments passed.
+   * @param args Expected arguments: [label(branch name)]
```
Comment (ratik21) [inline packages/algorand-js/src/interpreter/opcode-list.ts:1209]:
```suggestion
   * @param args Expected arguments: [label of branch]
```

File: packages/algorand-js/src/interpreter/opcode-list.ts (hunk 2 above)
Comment (amityadav0) [reply packages/algorand-js/src/interpreter/opcode-list.ts:200]: hmm, just checked TEAL opcode documentation `intcblock` and `bytecblock` are wrongly implementated.

Comment (amityadav0) [reply packages/algorand-js/src/interpreter/opcode-list.ts:200]: ** Discussion 
- On call discussed and checked both the opcodes, docs mentioned that we require a length for both, but when running teal codes, that's not the case.

**Review by ratik21 — APPROVED** [review]

**Pull request merged**